pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -937,6 +955,12 @@ in
     '';
   };
 
+  darkearth-nvim = super.darkearth-nvim.overrideAttrs {
+    dependencies = with self; [
+      lush-nvim
```

Comment:
general comment, I prefer self.lush-nvim when there is only one dependency. For future PRs.

Reply:
Yeah sorry, there's a bit of mixed usage in here that I wanted to cleanup in a general cleanup commit so I haven't been good about it